src/main/java/org/apache/commons/lang3/ArrayUtils.java

```diff
@@ -3254,6 +3254,57 @@ public static int indexOf(final Object[] array, final Object objectToFind, int s
         return INDEX_NOT_FOUND;
     }
 
+    /**
+     * <p>Finds the indices of the given object in the array.
+     *
+     * <p>This method returns an empty BitSet for a {@code null} input array.
+     *
+     * @param array  the array to search through for the object, may be {@code null}
+     * @param objectToFind  the object to find, may be {@code null}
+     * @return a BitSet of all the indices of the object within the array,
+     *  an empty BitSet if not found or {@code null} array input
```

Comment:
`@since` tag should be provided for newly provided methods.

Reply:
Okay, which version? 4.0?

Reply:
The next version is `3.10`.

Reply:
Done